testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -170,13 +180,59 @@ def uma_metadata(self, name, endpoint, credentials, **common_features):
 class ResponsesSection(Section, Responses):
     """Section which contains response configuration"""
 
+    def add(
+        self,
+        name: str,
+        value: dict,
+        wrapper_key: str = None,
+        wrapper: Literal["httpHeader", "envoyDynamicMetadata"] = None,
+        **common_features
```

Comment:
Keyword arguments here can be placed anywhere after the positional arguments. They are not dependent on their order as the new private method `add` is introduced in the `ResponsesSection` class. This method is used in every* other response section method instead of direct `add_item` inheritance.